base/sysimg.jl

```diff
@@ -583,7 +582,6 @@ end
     @deprecate_binding REPLCompletions root_module(Base, :REPL).REPLCompletions true ", use `REPL.REPLCompletions` instead"
     @deprecate_binding Terminals       root_module(Base, :REPL).Terminals       true ", use `REPL.Terminals` instead"
 
-    @deprecate_binding Pkg root_module(Base, :Pkg) true ", run `using Pkg` instead"
```

Comment:
Will doing this be a hard break for CI scripts that run things like `julia -e 'Pkg.thing()'`?

Reply:
Yes, I don't think we should do this until CI scripts are updated to import Pkg.

Reply:
I don't think we can have a deprecated binding exported from Base and have `using InteractiveUtils` import `Pkg` at the same time. I guess I'll try to figure out a way to make that happen though.

Reply:
Well, I'll be damned, it seems to work. I guess because both bindings are to the same thing.